// Advance 

console.log(one(5));            //this is not show any error

function one (num){
    return num +1;
}



// console.log(two(8));         This is show error 

const two = function(num) {
    return num+2;
}

